### DISCUSSION TITLE
remove (or move) paused prop

### DISCUSSION THREAD

**Discussion by YangJonghun** [TheWidlarzGroup/react-native-video#3576] [Ideas]
### What problem are you trying to solve?

I think the `paused` prop needs to be fixed before v6 is released.

Many situations require synchronizing the state of the video and the UI (controller) implemented in JS, and it would be great if we could only synchronize it in JS, but in reality the state is often changed by other external factors, so there is a lot of room for misunderstanding and misuse.

### Proposed solution

I think the proper implementation of paused is to expose it externally so that it is controllable only when the first state is injected, such as `startPosition` in the `source` prop.

This would remove (or move) the `paused` prop.

example)
```js
source={{ initPlaybackState?: "play" | "pause" }}
// or
source={{ playWhenReady?: boolean }}
```

### Alternatives considered

In most cases, you can synchronize the state in JS with `ref.resume()`, `ref.puase()`, and `onPlaybackStateChanged`, with the only exception being that they don't handle whether or not the video is playing when it first starts.

### Version line

v6

**Comment by freeboub** [comment]
@YangJonghun I hope you fix this issue, but you can use the prop at start pause={true}. and then use `ref.resume()`, `ref.pause()` afterward. maybe we can add a `onPlaybackStateChanged` near the `onload` event ?
